### PR TITLE
Blui 4369 migrate hero banner

### PR DESCRIPTION
**Pull request by JeffGreiner-eaton**
Fixes Blui-4369.

#### Changes proposed in this Pull Request:
- Migrate hero banner and test
- view example from https://github.com/etn-ccis/blui-react-native-showcase-demo/pull/97
- 

#### Screenshots / Screen Recording (if applicable)
-
image

#### To Test:
- 

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Review by ektaghag-eaton — COMMENTED** [review]
**Channel Value Props CSS:**
Icon -  color should be `#44474E`.
Value - lineHeight needs to be 28px and font-size 22px

**Hero prop CSS:**
Label - font-weight should be 500 , color should be #44474E

File: demos/showcase
Comment (ektaghag-eaton) [inline demos/showcase:1]: Do we need this file to be merged?

**Review by ektaghag-eaton — COMMENTED** [review]
Changes working fine. Logged an issue for Channel Value and Hero component.

File: demos/showcase
Comment (JeffGreiner-eaton) [reply demos/showcase:1]: not sure, is this just the pointer ?

**Review by daileytj — APPROVED** [review]

**Pull request merged**